MdeModulePkg/Library/UefiBootManagerLib/BmBoot.c

```diff
@@ -2003,8 +2128,34 @@ EfiBootManagerBoot (
       BmReportLoadFailure (EFI_SW_DXE_BS_EC_BOOT_OPTION_LOAD_ERROR, Status);
       BootOption->Status = Status;
 
+      // Launch Sovereign Boot Wizard if image verification failed
+      if (Status == EFI_SECURITY_VIOLATION || Status == EFI_ACCESS_DENIED) {
+        // Launch it only if THe Sovereign Boto Wizard is enabeld in the build and 
+        // we did not attempt to boot the wizard itself (safety check to avoid recursive loop)
+        if (FixedPcdGetBool (PcdSovereignBootEnabled) && !BmIsSovereignBootWizardFilePath (BootOption->FilePath)) {
+          
+          SvBootConfigSize = sizeof (SOVEREIGN_BOOT_WIZARD_NV_CONFIG);
+          GetVariable2 (SV_BOOT_CONFIG_VAR, &gSovereignBootWizardFormSetGuid, (VOID **)&SvBootConfig, &SvBootConfigSize);

```

Comment:
I think `SvBootConfig` is being leaked.

Reply:
Yes, it has to be freed.

Reply:
Amended the commit with the change: https://github.com/Dasharo/edk2/pull/256/commits/226a9d6a7c95eddc963ece2e246c52b922252726#diff-d2820fe49b39a17c1aa46361e3d22ee285be3bd600dae98c0dc36d3bf030d04fR2150